webpack.config.dist.js

```diff
@@ -0,0 +1,23 @@
+require("babel/register");
```

Comment:
Some of this is lazy because I have another PR sitting out there to reformat everything using `esformatter`. By the time I was writing this PR I assumed the formatting one would be merged already or that they would come at the same time.